platform/linux-generic/include-abi/odp/api/abi/packet_io_types.h

```diff
@@ -43,6 +43,8 @@ typedef struct odp_pktout_queue_t {
 #define ODP_PKTIO_INVALID _odp_cast_scalar(odp_pktio_t, 0)
 #define ODP_LSO_PROFILE_INVALID _odp_cast_scalar(odp_lso_profile_t, 0)
 
+#define ODP_PKTIO_MAX_INDEX 64
```

Comment:
Would a more suitable default actually be 63?

Reply:
True, changed in V2.